src/drivers/boards/aerofc-v1/aerofc_usb.c

```diff
@@ -78,13 +78,6 @@
 
 __EXPORT void stm32_usbinitialize(void)
```

Comment:
When I did this it was breaking the build in more ways than I wanted to fix. I'll check it again.